### PR TITLE
fix: Responsiveness issue of ratelimit page

### LINKED ISSUE #2409: [oss.gg hackathon][Bug]:Responsiveness issue on RateLimit page
### What side quest or challenge are you solving?

Screenshot 2024-10-14 at 2 52 50 PM

### Points

150

### Description

Steps to reproduce -
- Log into the application
- Navigate to Ratelimit page

### Provide proof that you've completed the task

/assign

### PR DESCRIPTION
**Pull request by BadriVishalPadhy**
## What does this PR do?

This fixes responsiveness problem in ratelimit page.

Fixes #2409

*If there is not an issue for this, please create one first. This is used to tracking purposes and also helps use understand why this PR exists*

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ x] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- Test A
- Test B

### Required

- [ x] Filled out the "How to test" section in this PR
- [ x] Read [Contributing Guide](./CONTRIBUTING.md)
- [ x] Self-reviewed my own code
- [ x] Commented on my code in hard-to-understand areas
- [ x] Ran `pnpm build`
- [ x] Ran `pnpm fmt`
- [ x] Checked for warnings, there are none
- [ x] Removed all `console.logs`
- [ x] Merged the latest changes from main onto my branch with `git pull origin main`
- [ x] My changes don't cause any responsiveness issues

### Appreciated

- [ x

https://github.com/user-attachments/assets/fee20428-cd73-47c7-bf76-16235b203e8a

] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the Unkey Docs if changes were necessary

## Release Notes

- **New Features**
	- Enhanced responsive design for the Ratelimit Namespace Page, improving visibility and layout on smaller screens.
  
- **Bug Fixes**
	- Adjusted layout behavior of the Empty Placeholder component for better width management and overflow prevention.

- **Style**
	- Corrected CSS class name in the Code component to ensure proper styling.

### CODE REVIEW

**Comment by BadriVishalPadhy** [comment]
@chronark

**Comment by BadriVishalPadhy** [comment]
@perkinsjr this wasn't required i guess 
-

**Comment by chronark** [comment]
cc @ogzhanolguncu

File: apps/dashboard/components/ui/code.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import type * as React from "react";
 import { cn } from "@/lib/utils";
 
 const codeVariants = cva(
-  "inlineflex font-mono items-center rounded-md border border-border bg-transparent px-2.5 py-2 text-sm   transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2",
+  "inlineflex font-mono items-center rounded-md border border-border bg-transparent text-sm   transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2",
```
Comment (ogzhanolguncu) [inline apps/dashboard/components/ui/code.tsx:7]: While we are at it lets fix this `inlineflex` typo as well.

**Comment by BadriVishalPadhy** [comment]
@ogzhanolguncu  Do checkout . . .

**Comment by ogzhanolguncu** [comment]
> @ogzhanolguncu Do checkout . . .

Great work @BadriVishalPadhy. If you are willing fix one more issue on this page after this is resolved:
From this:
image
To this:
Screenshot 2024-12-10 at 11 35 17

Feel free to open a ticket and do this. Don't forget to move separators as well don't do it like the SS 😄

**Review by ogzhanolguncu — APPROVED** [review]

**Comment by BadriVishalPadhy** [comment]
@ogzhanolguncu 






should it become vertically aligned  in  mobile view  or in all views . . .

**Pull request merged**